app/services/idv/user_bundle_tokenizer.rb

```diff
@@ -0,0 +1,43 @@
+module Idv
+  class UserBundleTokenizer
+    def initialize(user:, idv_session:, service_provider: nil)
+      @user = user
+      @idv_session = idv_session
+      @service_provider = service_provider
+    end
+
+    def call
```

Comment:
how would you feel about moving away from generic `call` methods to like `token`?
```suggestion
    def token
```

Reply:
Yup. Done in 04873b3c9